File: go/1041.go
package main

import "fmt"

func main() {
	var x, y float64
	fmt.Scanf("%f", &x)
	fmt.Scanf("%f", &y)

	if x == 0 && y == 0 {
		fmt.Printf("Origem\n")
		return
	}

	if x == 0 {
		fmt.Printf("Eixo Y\n")
		return
	}

	if y == 0 {
		fmt.Printf("Eixo X\n")
		return
	}

	if x > 0 && y > 0 {
		fmt.Printf("Q1\n")
		return
	}

	if x < 0 && y > 0 {
		fmt.Printf("Q2\n")
		return
	}

	if x < 0 && y < 0 {
		fmt.Printf("Q3\n")
		return
	}

	if x > 0 && y < 0 {
		fmt.Printf("Q4\n")
		return
	}
}

File: lua/1041.lua
local x = io.read("*n")
local y = io.read("*n")

if x == 0 and y == 0 then
    print("Origem")
    return
end

if x == 0 then
    print("Eixo Y")
    return
end

if y == 0 then
    print("Eixo X")
    return
end

if x > 0 and y > 0 then
    print("Q1")
    return
end

if x < 0 and y > 0 then
    print("Q2")
    return
end

if x < 0 and y < 0 then
    print("Q3")
    return
end

if x > 0 and y < 0 then
    print("Q4")
    return
end
